src/Pages/Login/LoginInput.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import LoginLinkBox from './LoginLinkBox';
+
+export default class LoginInput extends Component {
+  state = {
+    passwordSectetOnOff: false,
```

Comment:
```suggestion
    isPasswordSecretOn: false,
```
- boolean 타입일 경우에는 `is-` 네이밍이 좋을거 같아요